crates/forge/src/lib.rs

```diff
@@ -56,11 +56,13 @@ static BUILTINS: Lazy<Vec<&str>> = Lazy::new(|| {
 
 /// Configuration of the test runner
 #[derive(Deserialize, Debug, PartialEq)]
+#[allow(clippy::struct_excessive_bools)]
 pub struct RunnerConfig {
     test_name_filter: Option<String>,
     exact_match: bool,
     exit_first: bool,
-    fork_targets: Vec<ForkTarget>,
+    only_ignored: bool,
+    include_ignored: bool,
```

Comment:
I guess this could be changed to something like
```rust
// temp name
enum Run {
  All,
  OnlyIgnored,
  OnlyNonIgnored
}

pub struct RunnerConfig {
  // ...
  run: Run
  // ...
}
```

but I'm not sure if it is better at all.